src/app-config/auth0_shim.ts

```diff
@@ -0,0 +1,315 @@
+import atob from "atob";
```

Comment:
I don't hate this, but did you try using the regular auth0 lib as opposed to the auth0 spa?
https://github.com/architect-team/architect-cli/pull/171/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R19

https://auth0.github.io/auth0.js/global.html#buildAuthorizeUrl

Reply:
Also worth looking into 
https://auth0.github.io/auth0.js/global.html#renewAuth

Reply:
So after some digging both of these share the same problematic property: they're in the `auth0-js` library which is built for use in the browser. Neither this nor `auth0-spa-js` (not totally clear on why they have both) will work in our CLI.

Reply:
I did try calling a few different ways on the `auth0` library, the "universal" one that we have in the CLI already and none of these methods exist.

Reply:
That's weird - found this issue 
https://github.com/auth0/node-auth0/issues/173#issuecomment-381767911

Reply:
Looks like there is another npm lib called auth0-js which is confusing. Can you give that a try - looks like its meant for the browser, but maybe the Authentication class would work?
https://github.com/auth0/auth0.js#auth0authentication

Reply:
Ah, didn't see these come in. I gotta configure github notifications better.

Yes, I came across that same issue too. That indicated to me that they didn't support it this, right? The auth0 guy basically just says, "build your own".

And yeah, `auth0-js` was the library I mentioned above. I'm not sure we should be importing a non-universal library in the CLI even if one specific flow works. That opens us up to runtime issues.